docs/changes/per-environment-apis.md

```diff
@@ -31,3 +31,4 @@ In Vite v6, it is now possible to create any number of custom environments (`cli
 - `server.moduleGraph` -> [`environment.moduleGraph`](/guide/api-environment-instances#separate-module-graphs)
 - `server.transformRequest(url, ssr)` -> `environment.transformRequest(url)`
 - `server.warmupRequest(url, ssr)` -> `environment.warmupRequest(url)`
+- `server.reloadModule(module)` -> `environment.reloadModule(module)`
```

Comment:
If we add a new deprecation code, we likely also need to extend this list above: https://github.com/vitejs/vite/blob/e49f50599d852eec644e79b074b4648e2dff1e5d/docs/changes/per-environment-apis.md#L14-L19